x/liquidity/keeper/common_test.go

```diff
@@ -10,12 +10,14 @@ import (
 
 	"github.com/tendermint/liquidity/app"
 	"github.com/tendermint/liquidity/x/liquidity"
+	"github.com/tendermint/liquidity/x/liquidity/keeper"
 	"github.com/tendermint/liquidity/x/liquidity/types"
 )
 
 // createTestInput Returns a simapp with custom LiquidityKeeper
 // to avoid messing with the hooks.
 func createTestInput() (*app.LiquidityApp, sdk.Context) {
+	keeper.BatchLogicInvariantCheckFlag = true
```

Comment:
If the test code does not use `createTestInput()` and use `app.CreateTestInput()` instead, won't the flag be `false` by default?

Reply:
Sure, I moved the enabling code to `app.CreateTestInput()`